setup.cfg

```diff
@@ -1,5 +1,13 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
Hahaha, agree .. Well, maybe @vbuterin could tell you what is the right version for now.

But the PR is OK. : +1: